.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -26,10 +36,13 @@ relevant to every Crossplane pull request.
 I have:
 - [ ] Run `make reviewable` to ensure this PR is ready for review.
 - [ ] Ensured this PR contains a neat, self documenting set of commits.
-- [ ] Updated any relevant [documentation], [examples], or [release notes].
-- [ ] Updated the RBAC permissions in [`clusterrole.yaml`] to include any new types.
```

Comment:
This file still exists, and is frequently forgotten when adding new types. Can you provide some context around why you think it should not be in this checklist?

Reply:
yes, this check is still valid to me

Reply:
actually, i think these basic permissions would be exercised by the integration tests.  when the pod starts up and creates watches for all the types its controllers care about, the pod should crash immediately if we have't added any new necessary permissions.  I think our integration automation would catch this.

Reply:
Awesome :squid: